certora/README.md

```diff
@@ -24,12 +24,18 @@ Note: the compiled contracts may include loops related to handling strings from
 
 This is checked in [`ExternalCalls.spec`](specs/ExternalCalls.spec).
 
+### ERC20 Compliance
+
+This is checked in [`ERC20.spec`](specs/ERC20.spec).
+
 ## Verification architecture
 
 ### Folders and file structure
 
 The [`certora/specs`](specs) folder contains the following files:
 
-- [`ExternalCalls.spec`](specs/Reentrancy.spec) checks that the Morpho token implementation is reentrancy safe by ensuring that no function is making and external calls and, that the implementation is immutable as it doesn't perform any delegate call.
+- [`ExternalCalls.spec`](specs/Reentrancy.spec) checks that the Morpho token implementation is reentrancy safe by ensuring that no function is making and external calls and, that the implementation is immutable as it doesn't perform any delegate call;
```

Comment:
```suggestion
- [`ExternalCalls.spec`](specs/ExternalCalls.spec) checks that the Morpho token implementation is reentrancy safe by ensuring that no function is making and external calls and, that the implementation is immutable as it doesn't perform any delegate call;
```
oops just saw this one

Reply:
My bad, fixed in https://github.com/morpho-org/morpho-token-upgradeable/pull/91/commits/5d04121027d06d7995fdc93045c975deb5cbbf32.